lib/katello/tasks/upgrades/3.18/add_cvv_export_history_metadata.rb

```diff
@@ -0,0 +1,18 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.18' do
+      task :add_cvv_export_history_metadata => ['environment', 'check_ping'] do
+        smart_proxy = SmartProxy.pulp_primary!
+
+        Katello::ContentViewVersionExportHistory.includes(:content_view_version).where(metadata: nil).find_each do |export_history|
```

Comment:
Added a nil check for metadata here so that we don't always generate metadata on each upgrade...

Reply:
+1 - APJ